render-config.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
```

Comment:
Should this be under `/api`? Not sure there will be much of a need for this in the UI